source/expressions.tex

```diff
@@ -4662,6 +4662,9 @@
 of an unsigned quantity is computed by subtracting its value from $2^n$,
 where $n$ is the number of bits in the promoted operand. The type of the
 result is the type of the promoted operand.
+\begin{note}
+The result is the two's complement function of the operand.
```

Comment:
This should differ in only the operative word: the "considered as unsigned" still applies when we're thinking of this as a complement operation (rather than merely _commenting_ that negative numbers are represented by the two's complement in this system), and the result is not itself a function.

Reply:
"The sine of 0 is 0" vs "the sine applied to 0 is 0"... both work, don't they? Yes, "of" has two meanings (function declaration vs evaluation), but it's not _wrong_ to use for evaluation.

I don't quite understand the first part; could you suggest a complete note?

Reply:
That said, I'm happy to lose the notes entirely now that we got rid of "unary"!

Reply:
Yes, but no one would say "the error function of 0.5" but rather "the error function{,'s value, evaluated} at 0.5" because "function of" is confusing.  (It is funny, though, that even English has declaration-follows-use here.)

I just meant
> The result is the two's complement of the operand (both considered as unsigned).

Reply:
Oh, OK, and for ones' complement you'd say the same, just with "ones'" instead of "two's"?

Reply:
Yes, exactly.

Reply:
Ah yes, that's what you suggested above. PTAL.

Reply:
It looks good to me; you might want to check that no one has qualms about "considering" things as unsigned in a note (even though the actual bit behavior should be pretty obvious to all).

Reply:
@jensmaurer, @burblebee, @jfbastien: any qualms about "considering" things as unsigned?

Reply:
We still haven't resolved whether there are qualms!

Reply:
I'm fine with the phrasing. It's a note.